src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,83 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+from utils import remove_angle_brackets
+
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
+    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
+    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_clinical_subtype',  # subtype of a disorder
+}
+ROBOT_TEMPLATE_HEADER = {
+    'mondo_id': 'ID',
+    'ordo_id': 'A oboInOwl:hasDbXref',
+    'subset': 'A oboInOwl:inSubset',
```

Comment:
## Robot subheader
- [ ] @matentzn @twhetzel I think this is right, but please review

Reply:
Technically this has to be

```
ROBOT_TEMPLATE_HEADER = {
    'mondo_id': 'ID',
    'subset': 'A oboInOwl:inSubset',
    'ordo_id': '>A oboInOwl:hasDbXref',
```

The first two columns ensure the subset is created, the third column is provided as evidence.

Reply:
Just corrected above comment. `oboInOwl:source` not `oboInOwl:hasDbXref`

Reply:
Perfect! I'll update the code just to be consistent.
** There is no need to re-run the code and re-generate the ROBOT file though. Simple copy-paste in the headers was already done.